src/main/java/Duke.java

```diff
@@ -1,10 +1,209 @@
+import exceptions.DukeExceptions;
+import exceptions.DukeInvalidInput;
+import exceptions.DukeInvalidTodo;
+
+import java.io.*;
+import java.util.Objects;
+import java.time.LocalDate;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    public static void lineOne() {
+        System.out.println("*************************************************************************");
+    }
+
+    public static void lineTwo() {
+        System.out.println("|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|-|");
+    }
+
+    public static String makeDesc(String[] text, int len) {
+        String newText = "";
+        for (int i = 1; i < len; i++) {
+            newText = newText + text[i] + " ";
+        }
+        return newText.trim();
+    }
+
+    public static void writeToFile(ArrayList<Task> list) throws IOException {
+        try {
+            File f = new File("/Users/riakhaitan/iP/ip/data/duke.txt");
+            f.createNewFile();
+
+            boolean directory = f.getParentFile().mkdirs();
```

Comment:
Just a suggestion, perhaps you could use a name for this variable that is more boolean-sounding such as isDirectory? Good use of mkdir() here, creating a directory to store data